File: dart/3.dart
import "dart:io";
void main() {
  //3
  String num1 = stdin.readLineSync()!;
  String num2 = stdin.readLineSync()!;
  // add all together
  String total = "${double.parse(num1)*double.parse(num2)}";
  print("number converted to int is ${total}");
}

File: c/3.c
#include <stdio.h>

int main() {
  double oneNum;
  double twoNum;

  scanf("%lf", &oneNum);
  scanf("%lf", &twoNum);
  // times all together
  double sum = oneNum * twoNum;

  printf("Your total number is: %lf", sum); // %lf is for double

  return 0;
}
